ansible-role-rocky-requirements.yml

```diff
@@ -0,0 +1,4 @@
+- name: rcbops_openstack-ops
+  scm: git
+  src: https://github.com/rsoprivatecloud/openstack-ops
+  version: master
```

Comment:
I'm not a big fan of this being here since we're not maintaining it, but since this is part of a larger discussion I think we should allow it.  I just wanted to publicly note that this playbook does not follow the upstream standards and including it causes RPC-O to diverge from OSA significantly going forward.